rfcs/0000-rlp-status.md

```diff
@@ -0,0 +1,157 @@
+# RFC Template
+
+- Feature Name: `policy_status_states`
+- Start Date: 2023-02-03
+- RFC PR: [Kuadrant/architecture#0009](https://github.com/Kuadrant/architecture/pull/9)
+- Issue tracking: [Kuadrant/architecture#0000](https://github.com/Kuadrant/architecture/issues/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes a new design for the `RateLimitPolicy` Status definition and transitions.
+
+# Motivation
+[motivation]: #motivation
+
+At the time being, the `RateLimitPolicy` Status doesn't clearly and truthfully communicate the actual state of
+reconciliation and healthiness with its Operator managed services, mainly in this case the Rate Limit service `Limitador`.
+
+As a consequence, only misleading information is shared causing unexpected errors and flawed assumptions.
+
+The following are some issues addressing the before mentioned drawbacks:
+
+* https://github.com/Kuadrant/kuadrant-operator/issues/87
+* https://github.com/Kuadrant/kuadrant-operator/issues/96
+* https://github.com/Kuadrant/kuadrant-operator/issues/140
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+The following proposal it's divided in 3 stages, where each one could be applied/develop in order and would reflect valuable
+and accurate information with different degrees of acuity. The Policy CRD Status in the following diagrams, are simplified
+as states, which in the [Reference-level explanation](#reference-level-explanation) will be translated to the actual Status Conditions.
+
+## Stage 1
```

Comment:
I am struggling to visualise this in the resource. Is validiation a status condition type?
IE ```
conditions:
    - lastTransitionTime: "2019-10-22T16:29:24Z"
      status: "True"
      message: "object change observed"
      type: Accepted
    - lastTransitionTime: "2019-10-22T16:29:24Z"
      status: "True"
      message: "change has successfully been applied"
      type: Applied
```

Reply:
nvm I see it outlined below